tests/PHPStan/Type/ObjectTypeTest.php

```diff
@@ -99,12 +99,12 @@ public function dataIsSuperTypeOf(): array
 			6 => [
 				new ObjectType(\ArrayObject::class),
 				new ObjectType(\Traversable::class),
-				TrinaryLogic::createMaybe(),
+				TrinaryLogic::createNo(),
 			],
 			7 => [
 				new ObjectType(\Iterator::class),
 				new ObjectType(\Traversable::class),
-				TrinaryLogic::createMaybe(),
+				TrinaryLogic::createNo(),
```

Comment:
This change is definitely wrong. They're both interfaces, you can have an object that implements both.